src/store/item-store.js

```diff
@@ -69,6 +69,28 @@ class ItemStore extends Store {
     }
 
 }
+function getItemToEdit(itemAddress){
+    let result = readAddress(itemAddress);
+    if (result.type !== 'item') {
+        throw new Error('Address is not an item.');
+    }
+    return items[result.index];
+}
+function changeTimestamp(itemAddress, date){
+    let itemToChange = getItemToEdit(itemAddress);
+    let timestamp = moment.tz(date, 'America/Chicago');
+    let hour = parseInt(timestamp.format('H'));
+    let minute = parseInt(timestamp.format('m'));
+    // check for times past 4:50pm
+    if (hour > 16 || (hour === 16 && minute >= 50)) {
```

Comment:
This is the second time this checking is being done. We ought to move it into some sort of utility module.